packages/fast-layouts-react/src/utilities/breakpoint-tracker.ts

```diff
@@ -32,15 +37,16 @@ class BreakpointTracker {
 
     /**
      * Constructor for the BreakpointTracker component.
+     * @param defaultBreakpoint?: number - optional breakpoint that can be used instead of window.innerWidth
      */
     constructor() {
-        if (!canUseDOM()) {
-            return;
-        }
-
-        this.breakpoint = identifyBreakpoint(window.innerWidth, this._breakpoints);
+        this.breakpoint = canUseDOM()
+            ? identifyBreakpoint(window.innerWidth, this._breakpoints)
+            : this.defaultBreakpoint;
 
-        window.addEventListener("resize", this.requestFrame);
+        if (canUseDOM()) {
+            window.addEventListener("resize", this.requestFrame);
+        }
```

Comment:
We are using `canUseDOM()` twice, can we store this as a variable?
Either that or do something like:

```typescript
if (canUseDOM()) {
    this.breakpoint = identifyBreakpoint(window.innerWidth, this._breakpoints);
    window.addEventListener("resize", this.requestFrame);
} else {
    this.breakpoint = this.defaultBreakpoint;
}
```

Reply:
Yep I'll update to your if-else, that looks better than storing in a const